src/SprykerEco/Zed/Unzer/Business/Reader/UnzerReader.php

```diff
@@ -118,4 +113,66 @@ public function getPaymentUnzerTransactionByPaymentIdAndParticipantId(
                 ->findPaymentUnzerTransactionByPaymentIdAndParticipantId($paymentId, $transactionType, $participantId)
             ?? new PaymentUnzerTransactionTransfer();
     }
+
+    /**
+     * @param \Generated\Shared\Transfer\CustomerTransfer $customerTransfer
+     *
+     * @return \Generated\Shared\Transfer\UnzerCustomerTransfer|null
+     */
+    public function getUnzerCustomerTransferByCustomerTransfer(CustomerTransfer $customerTransfer): ?UnzerCustomerTransfer
+    {
+        return $this->unzerRepository->findUnzerCustomerByIdCustomer($customerTransfer->getIdCustomer());
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\UnzerCredentialsCriteriaTransfer $unzerCredentialsCriteriaTransfer
+     *
+     * @return \Generated\Shared\Transfer\UnzerCredentialsTransfer|null
+     */
+    public function getUnzerCredentialsByCriteria(UnzerCredentialsCriteriaTransfer $unzerCredentialsCriteriaTransfer): ?UnzerCredentialsTransfer
```

Comment:
Since it can return `null`
```suggestion
    public function findUnzerCredentialsByCriteria(UnzerCredentialsCriteriaTransfer $unzerCredentialsCriteriaTransfer): ?UnzerCredentialsTransfer
```